perf/fiotest.py

```diff
@@ -44,16 +46,34 @@ def setUp(self):
         """
         Build 'fio'.
         """
-        tarball = self.fetch_asset('http://brick.kernel.dk/snaps/fio-2.1.10.tar.gz')
+        sm = SoftwareManager()
+        dist = distro.detect()
+        if 'Ubuntu' in dist.name:
+            deps = ['libaio1', 'libaio-dev', 'bzip2']
+        else:
+            deps = ['libaio', 'libaio-devel', 'bzip2']
```

Comment:
At least in Fedora, `libaio` is a dependency for `libaio-devel`, so it will be installed anyway.
I'm in favour of now repeating code, so I think you could initialize `deps` with `['bzip2']` before this if and, inside the if, extend it with what is needed for each distro.